go/pkg/yolosvc/service.go

```diff
@@ -32,8 +33,9 @@ type Service interface {
 }
 
 type service struct {
-	startTime              time.Time
 	db                     *gorm.DB
```

Comment:
remove the old db pointer, it will help you detect that you forgot to port some code (i.e., tests)